Modules/Authentication/MultiAuthentication.cs

```diff
@@ -0,0 +1,31 @@
+﻿using GenHTTP.Api.Content;
+using GenHTTP.Modules.Authentication.Multi;
+
+namespace GenHTTP.Modules.Authentication;
+
+public static class MultiAuthentication
+{
+    #region Builder
+
+    /// <summary>
+    /// Creates a authentication handler that will use
+    /// underlying handlers to authenticate the request.
+    /// </summary>
+    public static MultiAuthenticationConcernBuilder Create() => new();
+
+    #endregion
+
+    #region Extensions
+
+    /// <summary>
+    /// Adds multi authentication to the handler.
+    /// </summary>
+    /// <param name="apiKeyAuth">The authentication concern to be added</param>
+    public static T Add<T>(this T builder, MultiAuthenticationConcernBuilder apiKeyAuth) where T : IHandlerBuilder<T>
```

Comment:
As `MultiAuthenticationConcernBuilder` is an `IConcernBuilder` this extension is not needed (as you can see in the coverage) and can safely be removed.